WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView.swift

```diff
@@ -1,4 +1,5 @@
 import SwiftUI
+import Yosemite
```

Comment:
I prefer to avoid adding production code solely for the use of tests... unfortunately, SwiftUI previews (which are essentially tests) push you to do that for convenience (!). Minimising the scope of that extra code keeps things healthier, as at least if it's inside the PreviewProvider, everyone knows that it's for tests.

The risk of adding this init is that future-us may think it's correct for use in production, and mistakenly use it when we don't need a completion handler. That would lead to us forgetting to log the country in our analytics.

An import like this isn't expensive though.

Reply:
That makes a lot of sense, thanks!